AdornUI-Stylesheets/navigation.css

```diff
@@ -0,0 +1,85 @@
+.navigation-container {
+  width: auto;
+  justify-content: space-around;
+  align-items: center;
+  background-color: var(--secondary-bg-color);
+}
+
+.navigation-item-2 {
+  width: 30rem;
+  height: 2rem;
+  border-radius: 10px;
+}
+
+.navigation-item-3 {
+  justify-content: space-around;
+  align-items: center;
+  margin: 1rem;
+  gap: 2rem;
+}
+
+.fa-bars {
+  display: none;
+}
+
+.search-bar {
+  font-size: 2rem;
+  background-color: var(--grey);
```

Comment:
avoid using names of color itself in variable. Use generic names like `--search-bar-bg-color` etc.